### ISSUE TITLE
Outdated vignette `modelsummary`

### ISSUE THREAD

**Issue by fkohrt** [vincentarelbundock/modelsummary#803]
The vignette `modelsummary` [currently reads](https://github.com/vincentarelbundock/modelsummary/blob/90be481639e9c34a04b904a4820cacb972c0cd6e/vignettes/modelsummary.qmd#L23)

> Before starting, we set `modelsummary` to use the `tinytable` package to draw table, and we set a few CSS properties to make tables more display in a more compact format on this website. This step is optional. 

But no such adjustments are made, so maybe remove it?

**Comment by vincentarelbundock** [comment]
Thanks a lot for the report. Should be fixed on the website now.

**Comment by vincentarelbundock** [comment]
That's no longer relevant because `tinytable` is default since version 2.0.0

**Comment by fkohrt** [comment]
Then I reckon the following bit can be removed from the Vignette as well?

> [...] we set `modelsummary` to use the `tinytable` package to draw table [...]
